### PR TITLE
Document `Engine.get_idle/physics_frames` methods

### PR DESCRIPTION
**Pull request by Xrayez**
As described in https://github.com/godotengine/godot/pull/35092#issuecomment-574040299.

### CODE REVIEW

File: doc/classes/Engine.xml [hunk 1]
```diff
@@ -51,6 +51,13 @@
 				Returns the frames per second of the running game.
 			</description>
 		</method>
+		<method name="get_idle_frames" qualifiers="const">
+			<return type="int">
+			</return>
+			<description>
+				Returns the total number of frames passed since engine initialization which is advanced on each [b]idle frame[/b], regardless of whether render loop is enabled, see [method get_frames_drawn].
```
Comment (akien-mga) [inline doc/classes/Engine.xml]: "whether **the** render loop"
I'd also do `s/, see/. See also/`.

Comment (Xrayez) [reply doc/classes/Engine.xml]: Replaced with `See also`. Not sure what you mean by `/`.

Comment (akien-mga) [reply doc/classes/Engine.xml]: That was `sed` syntax for a substitution :)
`s/a/b/` means "replace 'a' by 'b'".

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**